packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -420,17 +473,27 @@ export default {
     onCaptchaVerified(response) {
       this.validationService.resetFormField("captcha", {
         reset: true,
-        value: response,
+        value: response
       });
     },
     onCaptchaExpired() {
       this.$refs.recaptcha.reset();
       this.validationService.resetFormField("captcha", {
         reset: true,
-        value: "",
+        value: ""
       });
       this.validationService.checkFormInput("captcha");
     },
+
+    toggleModel() {
+      this.isShowModel = !this.isShowModel;
+    },
+    setTacUrlAndRedirect() {
+      const dataStamp = Date.now();
+      const acceptContactValue = document.getElementById("okta-contact").checked;
+      const oktaTaCUrl = this.socialUrl + `?okta_AcceptedToS=${acceptContactValue}&okta_ts_AcceptedToS=${dataStamp}`;
+      //window.location.href = oktaTaCUrl;
```

Comment:
🚧  Are we awaiting the URL schema to be finalized? We'll need to just remember to block merging until this is resolved. This comment can serve as that blocker. 🚧

Reply:
https://oktainc.atlassian.net/browse/OKTA-368714

Reply:
Got it! That merged this morning so it will not be available until next release 2020.03.1, which should be out on 2021-03-16

Will social login still work if we use these parameters in the URL before the change is released, or do we have to hold this change until the monolith release is deployed to OK12?

Reply:
I'm not sure what should I get in case of successfully complete social auth but I see this page
<img width="825" alt="Screenshot at Mar 01 18-21-24" src="https://user-images.githubusercontent.com/71866734/109526413-33b67280-7abb-11eb-92d2-e3b1c3097da8.png">

Reply:
Yeah it will still succeed but some parameters in the social URL will be ignored by the backend. We need to make sure the backend is actually accepting the parameters before we can show the modal to the user.